installers/aws-docker/install/run-gitlab.sh

```diff
@@ -11,22 +17,17 @@ CERT_FILE=/etc/gitlab/ssl/$HOST_DNS_NAME".crt"
 GITLAB_DATA_DIR=$CONTAINER_WORK_DIR/gitlab
 mkdir -p $GITLAB_DATA_DIR
 
-build()
-{
-	$pwd/gitlab/build.sh
-}
-
 start()
 {
-	build 
-
 	cp -r $CERT_DIR/ssl $GITLAB_DATA_DIR
 
 	docker run --name gitlab --env \
 		GITLAB_OMNIBUS_CONFIG="external_url 'https://$HOST_DNS_NAME:6443/'; gitlab_rails['lfs_enabled'] = true; nginx['listen_port'] =443 ; nginx['ssl_certificate'] = '$CERT_FILE' ; nginx['ssl_certificate_key'] = '$KEY_FILE'; letsencrypt['enable'] = false ; gitlab_rails['initial_root_password'] = '$GITLAB_ROOT_PASSWORD';" \
+		--env \
+		GITLAB_POST_RECONFIGURE_SCRIPT="gitlab-rails runner 'ApplicationSetting.last.update_attributes(signup_enabled: false)'" \
 		--publish 6443:443 \
 		-v $GITLAB_DATA_DIR:/etc/gitlab \
-		gitlab-ce-custom:latest
+		gitlab-ce:latest
```

Comment:
This has to be the Dockerhub image name.  gitlab/gitlab-ce:latest and not gitlab-ce:latest

Reply:
Thanks for spotting this! Looks like I had a `gitlab-ce` image locally and didn't notice.